libs/payments/src/lib/payment-provider/bexio/bexioUtils.ts

```diff
@@ -0,0 +1,47 @@
+import {MemberPlan, PaymentState, Subscription, User} from '@prisma/client'
+import Bexio, {ContactsStatic} from 'bexio'
+import {MappedReplacer} from 'mapped-replacer/dist/types'
+
+const {ContactSearchParameters} = ContactsStatic
+
+export type BexioInvoiceStatus = 7 | 8 | 9 | 16 | 19 | 31
```

Comment:
Could use an Enum and then give them names, makes the status clearer and no need for comments in the switch statements

Reply:
that was my thought, but in enum a member cannot have a numeric name